frontend/www/js/omegaup/components/course/ViewProgress.vue

```diff
@@ -151,19 +151,26 @@ function toOds(courseName: string, table: string[][]): string {
 export default class CourseViewProgress extends Vue {
   @Prop() assignments!: omegaup.Assignment[];
   @Prop() course!: types.CourseDetails;
-  @Prop() students!: omegaup.CourseStudent[];
+  @Prop() students!: types.StudentProgress[];
 
   T = T;
 
   score(
-    student: omegaup.CourseStudent,
+    student: types.StudentProgress,
     assignment: omegaup.Assignment,
   ): number {
-    let score = student.progress[assignment.alias] || 0;
-    return parseFloat(String(score));
+    if (!student.progress.hasOwnProperty(assignment.alias)) {
+      return 0;
+    }
+
+    let score = 0;
+    for (let problem in student.progress[assignment.alias]) {
+      score += student.progress[assignment.alias][problem];
+    }
+    return parseFloat(score.toString());
```

Comment:
```suggestion
    return score;
```

`score` is already a float (technically a [`Number`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) since JavaScript / TypeScript has a single type for all integers / floats / doubles), so there's no point in roundtripping from float to string to float.